model/src/main/java/oracle/kubernetes/weblogic/domain/v1/DomainSpec.java

```diff
@@ -847,6 +930,7 @@ public String toString() {
           .append("imagePullSecrets", imagePullSecrets)
```

Comment:
Need to add logHome to toString(), hashCode() and equals() as well.

Reply:
@alai8 Anthony, has this been done?

Reply:
yes, they are added to v2/DomainSpec.java

Reply:
thanks